README.md

```diff
@@ -72,6 +88,36 @@ Any other Airflow template variables will not work in `airflow.yml`.
 for the library to digest, but also automates Docker image building and pushes generated directory to the cloud storage
 of your choice.
 
+## Development
+
+### Running Tests
+
+```bash
+# Install with test dependencies
+pip install -e ".[tests]"
+
+# Run tests
+pytest tests/
+
+# Run with coverage
+pytest tests/ --cov=dbt_airflow_factory --cov-report=term-missing
+```
+
+### Known Issue: Installing on Some Systems
+
+If you encounter compilation errors related to `google-re2` (an Airflow dependency):
```

Comment:
This was fixed on current lib release by adding `--only-binary=google-re2` option during `pip install`